testcafe/tests/photos/photos_crud.js

```diff
@@ -1,40 +1,71 @@
 import { photosUser } from '../helpers/roles' //import roles for login
-import { TESTCAFE_PHOTOS_URL } from '../helpers/utils'
+import { TESTCAFE_PHOTOS_URL, SLUG } from '../helpers/utils'
 import random from 'lodash/random'
+import { VisualReviewTestcafe } from '../helpers/visualreview-utils'
 import Viewer from '../pages/photos-viewer/photos-viewer-model'
 import Timeline from '../pages/photos/photos-timeline-model'
 
 const timelinePage = new Timeline()
 const photoViewer = new Viewer()
 
-fixture`PHOTOS - CRUD`.page`${TESTCAFE_PHOTOS_URL}/`.beforeEach(async t => {
-  console.group(`\n↳ ℹ️  Loggin & Initialization`)
-  await t.useRole(photosUser)
-  await timelinePage.waitForLoading()
-  await timelinePage.initPhotosCount()
-  console.groupEnd()
-})
+//Scenario const
+const FEATURE_PREFIX = 'PhotosCrud'
+const FIXTURE_INIT = `${FEATURE_PREFIX} 1- Photos Navigation`
+const TEST_SELECT1 = `1-1 Select 1 photo`
+const TEST_SELECT2 = `1-2 Select 3 photos`
+const TEST_VIEWER_FIRST = `1-1 Open viewer for 1st photo`
+const TEST_VIEWER_LAST = `1-1 Open viewer for last photo`
+const TEST_VIEWER_OTHER = `1-1 Open viewer for 2nd photo`
 
-test('Select 1 pic from Photos view', async () => {
-  console.group('↳ ℹ️  Select 1 pic from Photos view')
+fixture`${FIXTURE_INIT}`.page`${TESTCAFE_PHOTOS_URL}/`
+  .before(async ctx => {
+    ctx.vr = new VisualReviewTestcafe({
+      projectName: `${SLUG}`,
```

Comment:
you don't need \`${}\` for those two lines

Reply:
Actually this is the old method for initializing VR !
Thanks for pointing it out!
Replace by 
```
   .before(async ctx => {
    await initVR(ctx, SLUG, FIXTURE_INIT)
  })
```